### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by clehner** [ssbc/patchwork#380]
I was publishing patchwork@v2.8.0 but when testing it found the following errors which break the UI:
- Minified exception error
- Error in SimpleInfinite
- Errors render below the main content, as seen in the screenshots below
- Clicking links doesn't work and instead trigger more of these two errors

```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
    at invariant (http://localhost:7777/main.build.js:29160:15)
    at Object.ReactOwner.addComponentAsRefTo (http://localhost:7777/main.build.js:23072:365)
    at attachRef (http://localhost:7777/main.build.js:23882:16)
    at Object.ReactRef.attachRefs (http://localhost:7777/main.build.js:23901:5)
    at ReactDOMComponent.attachRefs (http://localhost:7777/main.build.js:23772:12)
    at CallbackQueue.assign.notifyAll (http://localhost:7777/main.build.js:11638:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (http://localhost:7777/main.build.js:23680:26)
    at ReactReconcileTransaction.Mixin.closeAll (http://localhost:7777/main.build.js:26509:25)
    at ReactReconcileTransaction.Mixin.perform (http://localhost:7777/main.build.js:26456:16)
    at batchedMountComponentIntoNode (http://localhost:7777/main.build.js:21692:15)
```
```
TypeError: Cannot read property 'offsetHeight' of undefined
    at SimpleInfinite._this.onScroll (http://localhost:7777/main.build.js:54425:13)
    at Object.invokeGuardedCallback (http://localhost:7777/main.build.js:20472:12)
    at executeDispatch (http://localhost:7777/main.build.js:13576:21)
    at Object.executeDispatchesInOrder (http://localhost:7777/main.build.js:13599:5)
    at executeDispatchesAndRelease (http://localhost:7777/main.build.js:13039:22)
    at executeDispatchesAndReleaseTopLevel (http://localhost:7777/main.build.js:13050:10)
    at Array.forEach (native)
    at forEachAccumulated (http://localhost:7777/main.build.js:26961:9)
    at Object.EventPluginHub.processEventQueue (http://localhost:7777/main.build.js:13255:7)
    at runEventQueueInBatch (http://localhost:7777/main.build.js:20541:18)
```

![2016-05-08-105615_1280x800_scrot](https://cloud.githubusercontent.com/assets/95347/15098425/9c9e4f22-150b-11e6-9a47-d52f8c420e75.png)
![2016-05-08-105622_1280x800_scrot](https://cloud.githubusercontent.com/assets/95347/15098426/a53f12ec-150b-11e6-8bb1-737bb5bd716d.png)

@pfraze does any of this look familiar?

**Comment by pfraze** [comment]
I just pulled from master, reinstalled the deps, and ran build-release, and it worked fine. Did you do anything different?

**Comment by clehner** [comment]
This was both in my dev installation with npm-linked dependencies and in my patchwork-electron/app installation where dependencies were updated with a `npm i`. I am trying reinstalling all dependencies now.

**Comment by pfraze** [comment]
what node version are you using?

**Comment by clehner** [comment]
node v5.6.0 for patchwork-electron, node v4.3.1 for dev. I just finished replacing node_modules for dev and that made the problem go away the issue. So I guess either some dep was not up-to-date or some old file was leftover from previous versions and interfering with the build. Continuing with the release now

**Comment by pfraze** [comment]
Ok

**Comment by clehner** [comment]
Something had gone wrong with the build step and so the main.build.js I published in ssb-patchwork@2.8.0 was bad. I've unpublished that am publishing 2.8.1 now which seems to be working.
